modules/seqexec/server/src/main/scala/seqexec/server/ghost/GHOSTController.scala

```diff
@@ -39,37 +39,61 @@ final case class GHOSTController[F[_]: Sync](ghostClient: GHOSTClient[F],
     def cfg[P: Show](paramName: String, paramVal: P) =
       Configuration.single(s"${ifuNum.ifuStr}.$paramName", paramVal)
 
-    val config = for {
-      _   <- nameOpt // We only do this because we want Configuration.Zero if there is no target name.
-      ra  <- raOpt
-      dec <- decOpt
-    } yield {
-      cfg("target", IFUTargetType.determineType(nameOpt).targetType) |+|
-        cfg("ra", ra.toDoubleDegrees) |+|
-        cfg("dec", dec.toDoubleDegrees) |+|
-        cfg("bundle", bundleConfig.configName)
+    (nameOpt, raOpt, decOpt) match {
+      // Case 1: IFU is in use here for an actual position.
+      case (Some(name@_), Some(ra), Some(dec)) =>
```

Comment:
isn't this the same as `Some(name)`?

Reply:
Yes, but I wasn't allowed to do that because I wasn't using `name`, so the compiler insisted I do this instead, IIRC.

Reply:
I guess I could instead do:
`case Some(ra), Some(dec) if nameOpt.isDefined() => ...`

Reply:
Dosen't `case (Some(name), Some(ra), Some(dec))` work?

Reply:
Ah, you're not using it,then `case (Some(_), Some(ra), Some(dec))`

Reply:
Maybe just `case (Some(_), Some(ra), Some(dec)) =>`?

Reply:
I suppose what would be easier (and works) is to just do `Some(_), Some(ra), Some(dec)`

Reply:
This seems like it would make the most sense to me, though:

```
case (Some(ra), Some(dec)) if nameOpt.isDefined =>
```

Reply:
But that doesn't match `(nameOpt, raOpt, decOpt)` ...

Reply:
No, I'd have to change the match to `(raOpt, decOpt)`, but the effect would be the same otherwise, no?